go/apps/http_api_nostream/vumi_app.py

```diff
@@ -271,12 +293,51 @@ def send_event_to_client(self, event, conversation, push_url):
                 "push_event_url not configured for conversation: %s" % (
                     conversation.key))
             return
-        return self.push(push_url, event)
+        return self.push(conversation.user_account.key, push_url, event)
 
     @inlineCallbacks
-    def push(self, url, vumi_message):
-        config = self.get_static_config()
+    def schedule_push_retry(self, user_account_key, url, method, data,
+                            headers):
+        if self.http_retry_api is None:
+            returnValue(None)
+        list_headers = dict((k, [v]) for k, v in headers.iteritems())
+        retry_headers = {
+            'Content-Type': 'application/json; charset=utf-8',
+            'X-Owner-ID': user_account_key.encode("utf-8"),
+        }
+        retry_data = json.dumps({
+            "intervals": self.http_retry_intervals,
+            "request": {
+                "url": url,
+                "method": method,
+                "body": data,
+                "headers": list_headers,
+            },
+        }).encode('utf-8')
+        try:
+            resp = yield http_request_full(
+                self.http_retry_api.encode("utf-8"), data=retry_data,
```

Comment:
From the config field description, `http_retry_api` is the base url. If that's the case, we should add `/requests/` to the url here.